biorbd_optim/acados_interface.py

```diff
@@ -130,70 +154,39 @@ def __prepare_acados(self, ocp):
                         raise RuntimeError("The objective function is not Lagrange nor Mayer.")
```

Comment:
Maybe it is just GithHub that has a weird visual here, but it seems that the indentation is messed up before this raise?